runelite-client/src/main/java/net/runelite/client/plugins/loottracker/LootTrackerBox.java

```diff
@@ -227,6 +229,13 @@ void rebuild()
 
 		validate();
 		repaint();
+		for (LootTrackerRecord rec: records)
+		{
+			if (rec.isShouldCollapseBox())
+			{
+				this.collapse();
```

Comment:
this can't be correct due to collapse re-looping records

Reply:
Would adding a break here fix that issue? since once we find a collapsed record all the records for the box are updated to collapse so we can stop checking the remaining records.